orion/pkg/controllers/publish_controller_test.go

```diff
@@ -38,3 +65,62 @@ func TestGetUnpublishedAchievements_Success(t *testing.T) {
 	assert.EqualValues(t, "message2", unpublishedDomains.Achieves[1].Message)
 	assert.EqualValues(t, 2, len(unpublishedDomains.Achieves))
 }
+
+func TestGetAllUnpublishedPrograms_Success(t *testing.T) {
+	SetupMock()
+
+	// Create new HTTP request to endpoint
```

Comment:
If this test is split into 6, this logic is duplicated 6 times. I think this should either be one test or there should be a helper method that gets the unpublishedDomains once.

Reply:
That's a good idea.